packages/vue/src/sdk.ts

```diff
@@ -0,0 +1,416 @@
+/* eslint-disable max-lines */
```

Comment:
lol, I had no clue we have such a thing. I'd remove it altogether from eslint config.